### DISCUSSION TITLE
Ceiling vent

### DISCUSSION THREAD

**Discussion by LucilePl** [firemodels/cfast#2128] [Q&A]
Hello,
In the simulations I made on CFAST, all the fumes are evacuated via the ceiling wind and no fresh air passes through it. However, manufacturer data usually indicates that 50% of the flow through the hopper corresponds to fumes and the rest corresponds to fresh air. Is there a way to change that and impose that 50% ?

Thank you in advance!

**Comment by preneke** [comment]
CFAST calculates where the air is taken from internally and it can't be set. Air can come from both layers in certain situations. I am not sure of this you will have to look in the Tech Ref to get the details but again the short answer is you can't determine where the outflow of a ceiling vent comes from.
